### DISCUSSION TITLE
grand-total `pdu -c`

### DISCUSSION THREAD

**Discussion by mcguirepatr** [KSXGitHub/parallel-disk-usage#149] [Ideas]
Is there a way to print the grand-total, like `du -c *` does? I would like to do a `pdu -sch *`, so that it prints the size of each directory, followed by a grand-total at the end. If not, then can this feature be added?
Patrick

**Comment by KSXGitHub** [comment]
Try the following:

```sh
pdu --max-depth=1 *
```
```sh
pdu --max-depth=2 *
```

If you are still not satisfied, please let me know the difference.

**Reply by mcguirepatr** [reply]
I tried both of those, and neither of them prints a grand total at the end.
Patrick

**Reply by KSXGitHub** [reply]
What do you mean? `pdu` always print a tree structure. The root of this tree should be equal to grand total.

**Comment by mcguirepatr** [comment]
Maybe I am not using your `pdu`. I am using a `pdu` from PURE Storage in May 2018.
Patrick

**Reply by KSXGitHub** [reply]
If you're using my `pdu`, then it should print a result that looks like this:

![sample pdu output](https://user-images.githubusercontent.com/11488886/127254941-d1fb30d8-18e0-40ac-a212-bbd6463aa624.png)

BTW, what is the `pdu` from PURE Storage that you speak of?

**Reply by mcguirepatr** [reply]
yes, the output is certainly different from what you show.
I don't know much about the `pdu` from PURE storage, except that the man page says May 2018.
Patrick

**Reply by KSXGitHub** [reply]
Did you install that `pdu` from a Linux distro? What distro is it? (Just curious)

BTW, my `pdu` stands for "parallel-disk-usage". And I guess that `pdu` stands for "PURE's Disk Usage"?

**Reply by mcguirepatr** [reply]
The PURE Storage stuff was already installed on the computer that I am using.  

I think the `p` in `pdu` might stand for `parallelizable`. There are a bunch of PURE Storage commands available, like `pls`, which has the man page title `pls - Utility to parallelize an ls call to an NFS server`, and for `pdu` the page title is `pdu - Utility to parallelize a du call to an NFS server`. Some of the other PURE Storage commands are  `pchmod`,  `pchown`,  `pfind`,  and `prm`. I just found out about the PURE Storage `pdu` today, and I do like it better than `du`.
Patrick

**Reply by KSXGitHub** [reply]
I searched "pdu - Utility to parallelize a du call to an NFS server" on Google and found no relevant result, lol. Anyway, I hope you can find the right issue tracker soon.

**Reply by mcguirepatr** [reply]
I just searched for "pure storage pdu" on Google, and this was one of the hits:
https://support.purestorage.com/Solutions/Linux/Linux_Reference/RapidFileToolkit
Patrick
